seed/csharp-sdk/auth-environment-variables/snippet.json

```diff
@@ -0,0 +1,29 @@
+{
+    "types": {},
+    "endpoints": [
+        {
+            "example_identifier": null,
+            "id": {
+                "path": "/apiKey",
+                "method": "GET",
+                "identifier_override": "endpoint_service.getWithApiKey"
+            },
+            "snippet": {
+                "type": "typescript",
+                "client": "using SeedAuthEnvironmentVariables;\n\nvar client = new SeedAuthEnvironmentVariablesClient(\"X_ANOTHER_HEADER\", \"API_KEY\");\nawait client.Service.GetWithApiKeyAsync();\n"
```

Comment:
Should we expect the `API_KEY` value here if environment variables are enabled? This parameter is optional, right?

Reply:
great find, fixing